### DISCUSSION TITLE
The previews are suddendly blank

### DISCUSSION THREAD

**Discussion by JustLarsHere** [vercel/community#3696] [Help]
### Summary

The previews in pr's are suddendly blank not just for me, but other people too.
![image](https://github.com/vercel/community/assets/118752107/2c4a3def-8d06-4701-8a9c-0867a7d2eeb2)

### Example

https://blockly-for-discord-6kj0bup1w-larsishere.vercel.app/

### Steps to Reproduce

I have no idea why this happened or what steps led to this.
I have checked similar discussions, but while it's the same problem the fixes are very specific to their deployment and I can't apply them.

https://blockly-for-discord-6kj0bup1w-larsishere.vercel.app/

**Comment by amyegan** [comment]
Hey @LarsIsHere. Are you still seeing this problem on new deployments?

**Reply by JustLarsHere** [reply]
Sorry for not answering, I got it fixed, it was just one code line with a typo
